src/ui/components/cards/cardcomponent.js

```diff
@@ -64,11 +66,53 @@ export default class CardComponent extends Component {
     this.verticalKey = data.verticalKey;
   }
 
+  onMount () {
+    const rtfElement = DOM.query(this._container, '.js-yxt-rtfValue');
+    if (rtfElement) {
+      const fieldName = rtfElement.dataset.fieldName;
```

Comment:
could fieldName go inside _handleRtfClickAnalytics, or is this for a performance improvement?

Reply:
It's a small performance optimization to compute it once in the onMount as opposed to recomputing it each time the handler is called.